var/spack/repos/builtin/packages/py-multiprocess/package.py

```diff
@@ -12,10 +12,12 @@ class PyMultiprocess(PythonPackage):
     homepage = "https://github.com/uqfoundation/multiprocess"
     url = "https://pypi.io/packages/source/m/multiprocess/multiprocess-0.70.5.zip"
 
+    version('0.70.7', sha256='3394f1fbd0d87112690a877e49eb7917d851ee8d822294d522dd4deae12febdb')
```

Comment:
There is no `.zip` file for 0.70.7. You'll have to implement a `url_for_version` or something to select the `.tar.gz` file.